statics/js/components/topology.js

```diff
@@ -593,6 +597,10 @@ var TopologyComponent = {
       }
 
       if (self.isK8SEnabled()) {
+        // FIXME: Without() is not yet supported by backend so for now don't filter out istio
```

Comment:
without is added in backend from this patch https://github.com/skydive-project/skydive/pull/1521